backend/windmill-common/src/utils.rs

```diff
@@ -34,6 +35,10 @@ lazy_static::lazy_static! {
         .timeout(std::time::Duration::from_secs(20))
         .connect_timeout(std::time::Duration::from_secs(10))
         .build().unwrap();
+    pub static ref GIT_SEM_VERSION: Version = Version::parse(
+        // skip first `v` character.
+        GIT_VERSION.split_at(1).1
```

Comment:
@rubenfiszel only possible crash is here but is GIT_VERSION supposed to be empty ?